preprocess_observation_smogon.py

```diff
@@ -1,12 +1,12 @@
 """ preprocess_observation for pkmn.py. Kept in its own separate file in part because it is long. Functions like this generally exist for all games, but they differ a lot game to game. """
 import numpy as np
 from pokedex import pokedex
-
+from game_model import *
 
 # Todo in maybe December: Make this handle all the Pokemon and not just our favorites
 p1a_indices = {'Aggron': 0, 'Arceus': 1, 'Cacturne': 2, 'Dragonite': 3, 'Druddigon': 4, 'Uxie': 5}  #p1a is our AI, BloviatingBob
 p2a_indices = {'Houndoom': 0, 'Ledian': 1, 'Lugia': 2, 'Malamar': 3, 'Swellow': 4, 'Victreebel': 5} #right now, p2a is a mechanical Turk
-combinedIndices = {'Aggron': 809*2+0, 'Arceus': 809*2+1, 'Cacturne': 809*2+2, 'Dragonite': 809*2+3, 'Druddigon': 809*2+4, 'Uxie': 809*2+5,    'Houndoom': 809*2+6, 'Ledian': 809*2+7, 'Lugia': 809*2+8, 'Malamar': 809*2+9, 'Swellow': 809*2+10, 'Victreebel': 809*2+11}
+combinedIndices = {'Aggron': NUM_POKEMON*2+0, 'Arceus': NUM_POKEMON*2+1, 'Cacturne': NUM_POKEMON*2+2, 'Dragonite': NUM_POKEMON*2+3, 'Druddigon': NUM_POKEMON*2+4, 'Uxie': NUM_POKEMON*2+5,    'Houndoom': NUM_POKEMON*2+6, 'Ledian': NUM_POKEMON*2+7, 'Lugia': NUM_POKEMON*2+8, 'Malamar': NUM_POKEMON*2+9, 'Swellow': NUM_POKEMON*2+10, 'Victreebel': NUM_POKEMON*2+11}
```

Comment:
Ideally derive this from p1a_indices and p2a_indices, so you don't have to revise it every time those change.

Reply:
I think this issue is going to be resolved at the same time the comment ``A bit of a yellow flag'' gets resolved.